.github/workflows/pullrequest.yml

```
@@ -44,11 +44,20 @@ env:
+  pre-checks:
+    uses: ./.github/workflows/pre-checks.yml
+    secrets: inherit
+
   prepare:
     uses: ./.github/workflows/install.yml
+    needs:
+      - pre-checks
     secrets: inherit
     with:
       main_branch: ${{ github.base_ref }}
+      pre_release: false
```

actionlint (1.7.4)

input "pre_release" is typed as string by reusable workflow "./.github/workflows/install.yml". bool value cannot be assigned

(expression)